src/midcom/datamanager/extension/type/photo.php

```diff
@@ -60,7 +60,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         {
             $builder->add('title', 'text');
         }
+        $builder->add('delete', 'checkbox', array('attr' => array(
+            "class" => "org_openpsa_photo_checkbox"
```

Comment:
Please use midcom_datamanager as CSS "namespace", as this code is not specific to openpsa (the application)